gcloud-report/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,92 @@
+steps:
+
+# TODO: container image with golang+gcloud
+# - id: 'System & Unit Tests'
+#   name: 'golang:1.15'
+#   dir: '${_SAMPLE_DIR}'
+#   args: ['go', 'test', '-v', '.']
+#   env:
+#     - "GCLOUD_REPORT_BUCKET=${_BUCKET}"
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  dir: '${_SAMPLE_DIR}'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${SHORT_SHA}', '.'] # Tag docker image with git commit sha
+
+- id: 'Push Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${SHORT_SHA}']
+
+- id: 'Deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud-slim:$_CLOUDSDK_VERSION'
+  entrypoint: /bin/bash
+  args:
+  - '-c'
+  - | # Cloud Run Service name must be less than 63 characters
```

Comment:
This shouldn't be required anymore, but not necessary to change in this PR.

Reply:
That's great, I didn't notice this change. I'll go ahead and try switching to $BUILD_ID, if it works I'll keep it.